src/Microsoft.Health.Fhir.TemplateManagement.UnitTests/TestData/UserFolder/ADT_A01.liquid

```diff
@@ -0,0 +1,86 @@
+﻿{
```

Comment:
Can we directly use the main templates here -> `/data/Templates/Hl7v2/ADT_A01.liquid`?